interfaces/builtin/desktop.go

```diff
@@ -244,14 +270,12 @@ func (iface *desktopInterface) MountConnectedPlug(spec *mount.Specification, plu
 		})
 	}
 
-	/*
-		appId := "snap.pkg." + plug.Snap.Name()
-		spec.AddUserMount(mount.Entry{
-			Name: "$XDG_RUNTIME_DIR/doc/by-app/" + appId,
-			Dir: "$XDG_RUNTIME_DIR/doc",
-			Options: []string{"bind", "rw"},
-		})
-	*/
+	appId := "snap." + plug.Snap().Name()
+	spec.AddUserMountEntry(osutil.MountEntry{
```

Comment:
Can you add a comment explaining what this is doing. Specifically that the document portal doesn't know who's accessing the filesystem and that it needs per-app subdirectories to grok that. I think it should be captured as a comment as otherwise someone might optimise this away eventually.